### ISSUE TITLE
Clean up after downloading source

### ISSUE THREAD

**Issue by ChanceNCounter** [OpenVoiceOS/ovos_skill_manager#40]
osm just resolved a skill from the Mycroft marketplace using the zipfile. It should delete the zipfile afterward. There's not much use caching the compressed source after unpacking it.

**Comment by JarbasAl** [comment]
the zip/tar should be saved to xdg data directory

these will be used to detect changes in upstream, md5 can be compared on download to detect an update, even if branch is just "dev" and not version tagged, think of it like a secondary local versioning system. 

By always keeping at least 2 of these we can always revert to a known good version, even if it no longer exists on github

**Comment by ChanceNCounter** [comment]
Fair enough. Didn't realize that was in the version tagging, but it makes sense. "At least 2 of these" is an important point, too. Rotate for storage. In that case, should the filenames be adjusted? `dismissal-skill.chancencounter` saved `dismissal-skill.zip`, which could only be answered by `.old` (meh.) On the other hand, the zipfile on GitHub is *just* a version number, also unhelpful.

Also, it might make sense to store these someplace intended for source, rather than in the skills folder, but that's just imo
